drivers/hud_twenty_twenty_two_to_twenty_twenty_four/app/models/hud_twenty_twenty_two_to_twenty_twenty_four/current_living_situation/change_living_situation.rb

```diff
@@ -0,0 +1,64 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudTwentyTwentyTwoToTwentyTwentyFour::CurrentLivingSituation
+  class ChangeLivingSituation
+    LIVING_SITUATIONS = {
+      16 => 116,
+      1 => 101,
+      18 => 118,
+
+      15 => 215,
+      6 => 206,
+      7 => 207,
+      25 => 225,
+      4 => 204,
+      5 => 205,
+
+      2 => 302,
+      29 => 329,
+      14 => 314,
+      32 => 332,
+      12 => 312,
+      13 => 313,
+      27 => 227,
+      36 => 336,
+      35 => 335,
+
+      22 => 422,
+      23 => 423,
+      26 => 426,
+      10 => 410,
+      21 => 421,
+      11 => 411,
+
+      30 => 30,
+      17 => 17,
+      24 => 24,
+      37 => 37,
+      8 => 8,
+      9 => 9,
+      99 => 99,
+
+      28 => 435,
+      19 => 435,
+      31 => 435,
+      34 => 435,
+      20 => 435,
+      3 => 435,
+    }.freeze
```

Comment:
These appear to be duplicative of `drivers/hud_twenty_twenty_two_to_twenty_twenty_four/app/models/hud_twenty_twenty_two_to_twenty_twenty_four/aggregated_enrollment/change_living_situation.rb` Can we centralize that? Maybe in the structure file?

Reply:
Actually in HudUtility2024 probably makes more sense.  Something like `def cls_migration_for_2022_to_2024`

Reply:
I moved it to a concern in the migrator -- We shouldn't need it outside of migration, so HUDUtility seems like the wrong place to me.

Reply:
Thanks